src/workloads/networking/CommitLatency.yml

```diff
@@ -137,6 +420,46 @@ Actors:
       ReadMode: primary
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: CommitLatency_wmajority_majority_session
+  Type: CommitLatency
+  Threads: 1
+  Repeat: 500
+  Database: test
+  Phases:
+  - *Nop
```

Comment:
FWIW you can use

```yaml
Phases:
- Phase: 0..N
  Operation: Nop
```

Reply:
Ah cool. None of the existing workloads have that, so I assumed it wasn't implemented.

Is there also an open ticket for just doing

    Phases:
        - Phase: N

...and allowing to leave out the Nop phases completely? (Ie Nop would be default.)